tools/cldr-apps/js/src/esm/cldrAddAlt.mjs

```diff
@@ -52,18 +52,12 @@ async function addChosenAlt(xpstrid, alt, callbackFunction) {
     return;
   }
   const url = cldrAjax.makeApiUrl("xpath/alt", null);
-  const init = {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
-      Accept: "application/json",
-    },
-    body: JSON.stringify({
-      alt: alt,
-      localeId: localeId,
-      hexId: xpstrid,
-    }),
+  const data = {
+    alt: alt,
+    localeId: localeId,
+    hexId: xpstrid,
   };
+  const init = cldrAjax.makePostData(data);
```

Comment:
changes in this file are only for refactoring, to encapsulate POST, headers, stringify